client/src/pages/Login/Login.scss

```diff
@@ -0,0 +1,374 @@
+
+.login-bg-images {
+  position: relative;
+  overflow: hidden;
+  width: 100vw;
+  height: 150vh;
+  bottom: 100vh;
+  flex-direction: none;
+  justify-content: none;
+  align-items: none;
+}
+
+.ptero {
+  position: absolute;
+  top: 8vh;
+  left: 5vw;
+  width: 20vw;
+  z-index: 8;
+
+  animation: ptero-fly infinite 15s ease-in-out;
+}
+@keyframes ptero-fly {
+  0% {
+    transform: translateX(-10px) rotate(-5deg);
+  }
+  50% {
+    transform: translateX(10px) rotate(5deg);
+  }
+  100% {
+    transform: translateX(-10px) rotate(-5deg);
+  }
+}
+
+.brachio-left {
+  position: absolute;
+  bottom: 0;
+  width: 40vw;
+  z-index: 8;
+  left: 0;
+
+  animation: brachio-sway infinite 15s ease-in-out;
+}
+
+.brachio-right {
+  position: absolute;
+  bottom: 0;
+  z-index: 8;
+  width: 50vw;
+  right: -2vw;
+
+  animation: brachio-sway infinite 15s ease-in-out;
+}
+
+@keyframes brachio-sway {
+  0% {
+    transform: translateY(-5px) rotate(-1deg) scale(1);
+  }
+  50% {
+    transform: translateY(5px) rotate(1deg) scale(1.02);
+  }
+  100% {
+    transform: translateY(-5px) rotate(-1deg) scale(1);
+  }
+}
+
+.ground {
+  overflow: hidden;
+  position: absolute;
+  bottom: -20vh;
+  width: 100vw;
+  z-index: 7;
+
+  animation: ground-sway infinite 15s ease-in-out;
+}
+
+@keyframes ground-sway {
+  0% {
+    transform: scale(1);
+  }
+  50% {
+    transform: scale(1.02);
+  }
+  100% {
+    transform: scale(1);
+  }
+}
+
+.mountain-back {
+  position: absolute;
+  bottom: -50vh;
+  left: -25vw;
+  width: 80vw;
+  z-index: -10;
+}
+
+.mountain-front-left {
+  overflow: hidden;
+  position: absolute;
+  left: 0;
+  bottom: -10vh;
+  height: 100vh;
+}
+
+.mountain-front-right {
+  overflow: hidden;
+  position: absolute;
+  bottom: 0;
+  right: 0;
+  height: 140vh;
+  z-index: -1;
+}
+
+.mountain-mid {
+  position: absolute;
+  overflow: hidden;
+  bottom: -65vh;
+  width: 90vw;
+  left: 4vw;
+  z-index: 0;
+}
+
+@include devices(tablet) {
+  .login-bg-images {
+    height: 100vh;
+    width: 100vw;
+    overflow: hidden;
+  }
+
+  .ptero {
+    top: 25vh;
+    left: 10vw;
+    width: 20vw;
+  }
+
+  .brachio-left {
+    bottom: 12vw;
+    width: 25vw;
+    left: 0;
+  }
+
+  .brachio-right {
+    bottom: 22vw;
+    width: 40vw;
+    right: 0;
+  }
+
+  .ground {
+    bottom: 0;
+    width: 100vw;
+  }
+
+  .mountain-back {
+    bottom: 10vh;
+    left: -40vw;
+    width: 100vw;
+  }
+
+  .mountain-front-left {
+    left: 0;
+    bottom: -10vh;
+    height: 50vh;
+  }
+
+  .mountain-front-right {
+    bottom: 0;
+    right: -5vw;
+    height: 80vh;
+  }
+
+  .mountain-mid {
+    bottom: 0;
+    width: 90vw;
+    left: 8vw;
+  }
+}
+
+// FORGOT PASSWORD POPUP
+
+.forgot-password-popup {
+  position: absolute;
+  z-index: 9;
+
+  background: rgba(111, 30, 136, 0.75);
+  width: 100vw;
+  height: 150vh;
+
+  top: 0;
+
+  display: flex;
+  flex-direction: column;
+  // justify-content: center;
+  align-items: center;
+}
+
+// POPUP ANIMATION
```

Comment:
@jameskokoska more questions... I also added a popup animation when you open the popup, but I don't know how to animate the closing part, if you have any suggestions that would be great!

Reply:
Easiest way is to use transition
Transition will automatically reverse the animation when the style is removed

Reply:
i might be doing transition wrong because it doesn't seem to be working... but I'll look at last year's code!